back/lib/tasks/single_use/20240124_rename_most_voted_ideas_widget.rake

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+MULTILOC_TYPES = {
+  'MostVotedIdeasWidget' => 'MostReactedIdeasWidget'
```

Comment:
In the previous task, I made a typo, and so it wasn't renamed https://github.com/CitizenLabDotCo/citizenlab/blob/master/back/engines/commercial/multi_tenancy/lib/tasks/core/migrate_craftjson.rake#L13